LICENSE

```diff
@@ -1,4 +1,4 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+This work by Kyle R. Conway (https://kylerconway.com) is licensed under the Creative Commons Attribution-ShareAlike 4.0
```

Comment:
To clarify some of my earlier comments, one of the reasons we don't provide attribution in the LICENSE file is that the license applies to all work in the repository, regardless of the author/artist.  While you are the first person to contribute to the libseccomp-artwork repository (once again, thanks for all your help!), I imagine over time we will end up putting other things in the artwork repository.